pymemcache/client/base.py

```diff
@@ -803,7 +821,7 @@ def _store_cmd(self, name, values, expire, noreply, cas=None):
             extra += b' ' + cas
         if noreply:
             extra += b' noreply'
-        expire = six.text_type(expire).encode('ascii')
+        expire = six.text_type(expire).encode(encoding)
```

Comment:
This looks like a good (but separate) bug fix. Can you submit it as its own Pull Request so we can easily give it its own tests and changelog entry?

(including the related lines below)

Reply:
remcached only supports text protocol but not binary protocol, in order for the remcached extension to consume pymemcahe, the encoding has to be changed to 'utf-8', that's why I added param encoding. Do you think it's necessary to have a separate PR?

Reply:
Got it. I _do_ think it would be helpful to break this into three smaller (sequential) PRs:

1. The `encoding` addition.
2. The `flags` addition.
3. The "value helper".

Reply:
totally! Will do!